proto/api/v1/payment.proto

```diff
@@ -113,8 +113,18 @@ message PaymentCustomer {
   optional string phone_number = 12;
   // Balance actual balance of the customer
   optional int64 balance = 13;
-  // PayPerInvoice indicates if the user choose to pay per invoice
-  optional bool pay_per_invoice = 14;
+  // PaymentMethod indicates which way the user wants to pay
+  optional PaymentMethod payment_method = 14;
```

Comment:
Is the PaymentMethod really optional ?

Reply:
I am still debating if we use `PAYMENT_METHOD_UNSPECIFIED` for the case if the user has no PaymentMethod yet, or in an error state where the type can not be determined.   
Or should we add a case to the enum like `PAYMENT_METHOD_NOT_ADDED`.  
What do you think?

Reply:
Unspecified is in my opinion exactly the case when no payment method is given.